### PR TITLE
Add HanToken 

### PR DESCRIPTION
**Pull request by han-identity**
Add Han Token 

data.json
logo.svn

### CODE REVIEW

File: data/Han/data.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "HanChain",
+    "symbol": "HAN",
+    "decimals": 18,
+    "website": "https://paykhan.io",
+    "telegram": "https://t.me/hanchain_official",
+    "twitter": "https://twitter.com/HanIdentity",
+    "discord": "https://discord.gg/5gtfUuvJJX",
+    "medium": "https://medium.com/@HanIdentity",
+    "youtube": "https://www.youtube.com/channel/UCQPzdwU4KHlXO3srolte0Dg",
+    "facebook": "https://www.facebook.com/HanChainGlobalOfficial-101331419212206",
```
Comment (smartcontracts) [inline data/Han/data.json:11]: Currently only the fields "website" and "twitter" are accepted -- "telegram", "discord", "medium", and "facebook" are not. Please remove these fields and validation will pass.

Comment (han-identity) [reply data/Han/data.json]: remove fields thx

**Comment by smartcontracts** [comment]
Looks like you just need to verify the code for the token on Goerli. Once that's done, this PR should be good to go.

**Comment by han-identity** [comment]
> We detected an issue with the files added in this PR. Please see the reported error message below:
> 
> error: Han: instance: requires property "description" error: Han: instance: requires property "website"

Hi. Added description, website and twitter. Can I now send the HAN tokens to L2?

**Comment by han-identity** [comment]
> Looks like you just need to verify the code for the token on Goerli. Once that's done, this PR should be good to go.

Hi. Completed verify the code for the token on Goerli, added description, website and twitter into data.json. Can I now send the han token to l2?

**Comment by han-identity** [comment]
Hi,, Not done yet?

**Comment by han-identity** [comment]
Hi. Please answer my questions. It seems like it's taking too long

**Comment by han-identity** [comment]
Hi, Please give me a quick reply.

Ethereum token listed on the CoinGecko Token List
https://www.coingecko.com/en/coins/hanchain

**Comment by smartcontracts** [comment]
Hi @han-identity sorry for the delay here. We had some issues with the verification scripts for this repository and are currently recreating them. We will merge this PR asap.

**Pull request merged**